pcsx2/ImGui/ImGuiFullscreen.cpp

```diff
@@ -2091,17 +2096,15 @@ bool ImGuiFullscreen::HorizontalMenuItem(GSTexture* icon, const ImVec2& icon_uv0
 	dl->AddImage(reinterpret_cast<ImTextureID>(icon->GetNativeHandle()), icon_pos, icon_pos + ImVec2(icon_size, icon_size), icon_uv0, icon_uv1);
 
 	const std::pair<ImFont*, float> title_font = g_large_font;
-	const ImVec2 title_size = title_font.first->CalcTextSizeA(title_font.second, avail_width, 0.0f, title);
+	const ImVec2 title_size = title_font.first->CalcTextSizeA(title_font.second, size.x, 0.0f, title);
 	const ImVec2 title_pos =
 		ImVec2(bb.Min.x + (avail_width - title_size.x) * 0.5f, icon_pos.y + icon_size + LayoutScale(10.0f));
-	const ImVec4 title_bb = ImVec4(title_pos.x, title_pos.y, title_pos.x + title_size.x, title_pos.y + title_size.y);
 
-	dl->AddText(title_font.first, title_font.second, title_pos, ImGui::GetColorU32(ImGuiCol_Text), title, nullptr, 0.0f,
-		&title_bb);
+	dl->AddText(title_font.first, title_font.second, title_pos, ImGui::GetColorU32(ImGuiCol_Text), title, nullptr, 0.0f);
```

Comment:
This will no longer truncate oversized titles, instead continuing to draw them beyond the right side of the item.

Currently I think we only use these for the landing page menus, so it might not matter

Reply:
JP translation of "Desktop Mode" was going over for a while, and it was kind of terrible to truncate it.

I think it now fits with the text size adjustments and removal of padding, but I'd still rather display the whole string than truncate.